app/main.py

```diff
@@ -1,8 +1,37 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class, clean_mark, brand) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center, clean_power, average_rating, count_of_ratings) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        total_income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                total_income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(total_income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        price = car.comfort_class * (
+            self.clean_power - car.clean_mark
+        ) * self.average_rating / self.distance_from_city_center
```

Comment:
The calculation of the washing price is correct, but it could be improved for readability by breaking it into multiple lines or using intermediate variables as suggested in the checklist: 'If you have some long math, you can split it onto additional variables, or break after binary operations (not before - it cause the W504 errors)'.